test/test_distribution.py

```diff
@@ -32,6 +32,7 @@
     from funsor.pyro.convert import dist_to_funsor
 
 
+@pyest.mark.xfail(get_backend() == "jax", "flaky test")
```

Comment:
@fehiepsi I've been seeing nondeterministic test failures with the jax backend. They resolve when I rerun the tests

Reply:
Thanks for letting me know! I will keep an eye on this.